src/main/java/frc/robot/commands/LiftRobot.java

```diff
@@ -0,0 +1,48 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+
+/**
+ * An example command.  You can replace me with your own command.
+ */
+public class LiftRobot extends Command {
```

Comment:
What is this command supposed to do? Is it used to lift the robot up at the end for climbing onto the platform?

Reply:
That is what it is going to be, using pistons, I'm currently having 4 pistons allocated to this subsystem